src/fixed-footer-layout.tsx

```diff
@@ -105,44 +100,49 @@ const FixedFooterLayout: React.FC<Props> = ({
         );
 
         const transitionAwaiter = waitForSwitchTransitionToStart(checkDisplayShadow);
-        addPassiveEventListener(window, 'resize', checkDisplayShadow);
-        addPassiveEventListener(window, 'scroll', checkDisplayShadow);
+        const scrollEventTarget = getScrollEventTarget(scrollable);
+        addPassiveEventListener(scrollEventTarget, 'resize', checkDisplayShadow);
```

Comment:
resize event listener should be on window, you cannot listen to dom elements resize without a ResizeObserver

Reply:
not an issue for now. I'm not aware of a fixed footer usage where the scrollable parent is not the html element. In that case the scrollEventTarget is window.